packages/react-app/src/components/pages/Bounties/BountyAccordion/index.tsx

```diff
@@ -31,6 +31,8 @@ const BountyList = ({ bounties }: BountyListProps): JSX.Element => {
 						claimedBy={claimedBy}
 						reward={reward}
```

Comment:
Bountyboard now expects a customer id, but as it currently stands that doesn't do anything in the bounty accordion, so for now just pass an emptry strung rather than fancy typecasting